ci/travis/ci.sh

```diff
@@ -205,7 +205,7 @@ test_cpp() {
   bazel test //cpp:cluster_mode_test --test_arg=--external-cluster=true --test_arg=--redis-password="1234" \
     --test_arg=--ray-redis-password="1234"
   # run the cpp example
-  bazel run //cpp/example:example
+  cd cpp/example && bazel --nosystem_rc --nohome_rc run //:example
```

Comment:
What problem does `--nosystem_rc --nohome_rc` solve? Should we add these to the doc?

Reply:
Bazel options has been inherited  in CI images. But this test is an independent project for users. We need add "--nosystem_rc --nohome_rc" in the test script. But we don't need to add this to the doc because there is no problem in user side.

![image](https://user-images.githubusercontent.com/26714159/124858937-8d86f600-dfe1-11eb-9f61-144962e03cdd.png)